packages/code_excerpt_updater/mono_pkg.yaml

```diff
@@ -1,6 +1,6 @@
 # See https://github.com/google/mono_repo.dart for details on this file
 sdk:
-- 2.17.5
+- 2.18.1
```

Comment:
You can use `pubspec` as a value here and it'll grab whatever you have in the pubspec as the lower bound.

So you could switch the lower bound to `sdk: '>=2.18.1 <3.0.0'` – then you don't have to update in multiple places.

It's also good to test the lowest declared lower bound!

Reply:
I got sick of all of the duplicate updates. 😁